stratum/lib/security/credentials_manager.h

```diff
@@ -8,95 +9,67 @@
 #include <memory>
 #include <string>
 
-#include "absl/synchronization/mutex.h"
 #include "grpcpp/grpcpp.h"
+#include "grpcpp/security/server_credentials.h"
 #include "grpcpp/security/tls_credentials_options.h"
 #include "stratum/glue/status/status.h"
 #include "stratum/glue/status/statusor.h"
 
-namespace stratum {
-using TlsCredentialReloadInterface =
-    ::grpc::experimental::TlsCredentialReloadInterface;
-using TlsCredentialReloadArg = ::grpc::experimental::TlsCredentialReloadArg;
-using TlsCredentialReloadConfig =
-    ::grpc::experimental::TlsCredentialReloadConfig;
-using TlsKeyMaterialsConfig = ::grpc::experimental::TlsKeyMaterialsConfig;
-using TlsCredentialsOptions = ::grpc::experimental::TlsCredentialsOptions;
-using ::grpc::experimental::TlsServerCredentials;
-
-// CredentialsReloadInterface is an implementation of
-// the TlsCredentialReloadInterface which helps reloading gRPC server
-// credentials(private key and certifications)
-// The `Schedule` function will be called when gRPC server initialized or
-// a new connection is created.
-class CredentialsReloadInterface : public TlsCredentialReloadInterface {
- public:
-  ~CredentialsReloadInterface() override = default;
-  CredentialsReloadInterface(std::string pem_root_certs,
-                             std::string server_private_key,
-                             std::string server_cert);
-
-  // Public methods from TlsCredentialReloadInterface
-  int Schedule(TlsCredentialReloadArg* arg) override
-      LOCKS_EXCLUDED(credential_lock_);
-  void Cancel(TlsCredentialReloadArg* arg) override;
-
-  // Loads new credentials
-  ::util::Status LoadNewCredential(const std::string ca_cert,
-                                   const std::string cert,
-                                   const std::string key)
-      LOCKS_EXCLUDED(credential_lock_);
-
-  // CredentialsReloadInterface is neither copyable nor movable.
-  CredentialsReloadInterface(const CredentialsReloadInterface&) = delete;
-  CredentialsReloadInterface& operator=(const CredentialsReloadInterface&) =
-      delete;
+DECLARE_string(ca_cert_file);
```

Comment:
Introduced by me: 
Introducing the `DECALRE_string` statements here, so that all included files where `#include credentials_manager.h` will have these available. This is one of the recommended methods of using GFlags according to official documentation.